mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -436,7 +462,10 @@ pub(super) async fn init_light_client(
     };
 
     drop_mutability!(rpc_clients);
```

Comment:
This `drop_mutability` can be moved below `drop_mutability!(errors);` where mutable `rpc_clients` is not needed anymore.

Reply:
done